youtube_dl/extractor/tumblr.py

```diff
@@ -29,6 +29,34 @@ class TumblrIE(InfoExtractor):
             'thumbnail': 're:http://.*\.jpg',
         }
     }, {
+        'url': 'http://hdvideotest.tumblr.com/post/130323439814/test-description-for-my-hd-video',
+        'md5': '99a84522f60972bf064a0b80f87bcbb5',
+        'resolution': 'hd',
```

Comment:
Unless I'm missing something, this field is not being checked, it should go inside `'info_dict`'  instead.